src/main/java/lotto/domain/LottoPapers.java

```diff
@@ -1,34 +1,28 @@
 package lotto.domain;
 
-import lotto.util.LottoNumberUtil;
-
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
 
 public class LottoPapers {
-    private final List<LottoPaper> papers;
-
-    public LottoPapers(int paperCount) {
-        papers = makeAutoLottoNums(paperCount);
-    }
+    private final List<LottoPaper> lottoPapers;
 
-    private List<LottoPaper> makeAutoLottoNums(Integer paperCount) {
-        return IntStream.range(0, paperCount)
-                .mapToObj(value -> new LottoPaper(getAutoLottoNums()))
-                .collect(Collectors.toList());
+    public LottoPapers(List<LottoPaper> lottoPapers) {
+        this.lottoPapers = lottoPapers;
     }
 
-    private List<LottoNum> getAutoLottoNums() {
-        return LottoNumberUtil.getAutoLottoNums();
+    public LottoPapers addAll(LottoPapers lottoPapers) {
+        List<LottoPaper> papers = new ArrayList<>();
+        papers.addAll(this.getLottoPapers());
+        papers.addAll(lottoPapers.getLottoPapers());
```

Comment:
LottpPapers 내부에서 자기 자신의 데이터를 처리하고 있기 때문에 getter를 사용하지 않고
직접 접근하는게 좋아보여요 :)

```suggestion
    public LottoPapers addAll(LottoPapers autoLottoPapers) {
        List<LottoPaper> papers = new ArrayList<>();
        papers.addAll(lottoPapers);
        papers.addAll(autoLottoPapers.lottoPapers);
```